contracts/epochs/FlowDKG.cdc

```diff
@@ -119,6 +333,7 @@ access(all) contract FlowDKG {
 
         /// Posts a whiteboard message to the contract
         access(all) fun postMessage(_ content: String) {
+            // TODO: DKG enabled?
```

Comment:
It seems like the smart contract allows posting whiteboard messages and sending final submissions even when the DKG is disabled 🤔. I don't think this matters functionally because:
- the DKG cannot be ended (winning final submission retrieved) if it is disabled
- when enabled/started, all state including whiteboard messages and submissions are wiped

For clarity though, I think we should disallow interacting with the contract while it is disabled.

Reply:
That is fine with me. Feel free to add those pre-conditions to the functions if you want!